scripts/feature_tests/feature_test_vars.sh

```diff
@@ -2,4 +2,4 @@
 
 export NUM_OF_MASTER_REPLICAS="3"
 export NUM_OF_WORKER_REPLICAS="1"
-export KUBERNETES_VERSION="v1.22.1"
\ No newline at end of file
+export KUBERNETES_VERSION="v1.22.3"
```

Comment:
In feature test we cannot use the latest version since in nodereuse will need to upgrade from this version to latest 
```suggestion
export FROM_K8S_VERSION="v1.22.2"
export KUBERNETES_VERSION=${FROM_K8S_VERSION}
```

Reply:
+1 I am also wondering this.